packages/components/src/local/organisms/setting-forces/index.tsx

```diff
@@ -115,9 +115,21 @@ export const SettingForces: React.FC<PropTypes> = ({
       })
 
       // show message
-      attributeErrors.length > 0 && toggleModal('The attributes for some assets did not match with type details. These fixes have been applied: ' + attributeErrors.join())
+      const attrsbuteErrorList = attributeErrors.reduce((html, item) => {
+        html += `<li>${item}</li>`
+        return html
+      }, '')
+      attributeErrors.length > 0 && toggleModal(`The attributes for some assets did not match with type details. These fixes have been applied: <br/> ${attrsbuteErrorList}`)
 
       if (onSave) {
+        forcesData.some(force => {
+          if (force.uniqid === selectedForce.uniqid) {
+            // TODO: should loop each asset and assign only the `attributeValues` for each one instead of assign the whole asset to force?
```

Comment:
Good thinking @lebaphi  - could you do this improved processing please?

Reply:
okay @IanMayo , I will fix it in another PR 👍🏻